ballerina/Ballerina.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 org = "ballerina"
 name = "data.xmldata"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
shouldn't we do a 0.2 release with this?

Reply:
We are allowing what we disallowed previously hence no need to go for 0.2 right? I hope that is fine

Reply:
If you think, this as behavioural change and should do a `0.2` we can do that.

Reply:
Adding the new field alone is pretty a much a backward incompatible change, but since we've not done 1.0 yet, we don't have much of a choice and normal versioning rules may not apply. Since this can be somewhat of a significant change though, I also agree that it would be better to do 0.2.

Reply:
Concluded to include this in 1.0.0 with update lang-version 2201.10.0